modules/tensor_mechanics/src/materials/ComputeThermalExpansionEigenstrainBase.C

```diff
@@ -42,5 +42,5 @@ ComputeThermalExpansionEigenstrainBase::computeQpEigenstrain()
   _eigenstrain[_qp].addIa(thermal_strain);
 
   _deigenstrain_dT[_qp].zero();
-  _deigenstrain_dT[_qp].addIa(-instantaneous_cte);
```

Comment:
@dschwen This is the same change made in my other PR. We could close that one and just merge this one when it's ready, which is probably preferable because this PR adds tests that depend on the sign being right. Part of my purpose in submitting the other PR was just to see if I broke anything with the sign change, which I didn't.